lib/active_model_serializers/railtie.rb

```diff
@@ -30,8 +29,8 @@ class Railtie < Rails::Railtie
     # inheritable, we don't want to set it on `ActiveModel::Serializer` directly unless
     # we want *every* serializer to be considered cacheable, regardless of specifying
     # `cache # some options` in a serializer or not.
-    initializer 'active_model_serializers.caching' => :after_initialize do
-      ActiveModelSerializers.config.cache_store     = ActionController::Base.cache_store
+    initializer 'active_model_serializers.caching', :after => 'action_controller.set_configs' do
+      ActiveModelSerializers.config.cache_store     = Rails.configuration.action_controller.cache_store
```

Comment:
Is there any advantage to this over `Rails.cache` at this point? I've read through the init code in Rails, but you clearly understand it better than I.